tests/Unit/CrudPanel/CrudPanelTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Backpack\CRUD\Tests\Unit\CrudPanel;
+
+use Illuminate\Database\Eloquent\Builder;
+
+class CrudPanelTest extends BaseCrudPanelTest
+{
+    public function testSetModelFromModelClass()
+    {
+        $this->crudPanel->setModel(TestModel::class);
+
+        $this->assertEquals($this->model, $this->crudPanel->model);
+        $this->assertInstanceOf(TestModel::class, $this->crudPanel->model);
+        $this->assertInstanceOf(Builder::class, $this->crudPanel->query);
+    }
+
+    public function testSetModelFromModelClassName()
+    {
+        $this->crudPanel->setModel('\Backpack\CRUD\Tests\Unit\CrudPanel\TestModel');
```

Comment:
Err, this is repeated twice - maybe make that a variable?

Reply:
Sure, but I'll update this in the [crud panel, columns and fields PR](https://github.com/Laravel-Backpack/CRUD/pull/902). This one should only contain the buttons trait unit tests, once that one is merged.

Reply:
If that's not meant to be in this PR, it probably should just come out of it :)

Reply:
When that PR will get merged, the changes will be reflected here, i.e. only the buttons trait diff will show up. 
Unfortunately, GitHub doesn't allow PR dependencies, and I reeeally didn't want to put everything in a single PR.

Do you have any suggestion on how we can manage different PRs for related/dependent tasks?

Reply:
In an ideal world, we'd have a branch per thing; or something. It's messy and imperfect.

ATM I think we concentrate on getting stuff in.